### DISCUSSION TITLE
Docker Error: Database Initialization Error

### DISCUSSION THREAD

**Discussion by InformaticaGuttmann** [requarks/wiki#3178] [Help / Questions]
Hi,
I try to install wikijs with docker container and mysql 8.0 container in same server. When I execute wikijs container then in log appear this meesage.

`2021-03-04T17:16:02.582Z [MASTER] error: Database Initialization Error: create table `analytics` (`key` varchar(255) not null, `isEnabled` boolean not null default '0', `config` json not null) default character set utf8mb4 - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use
near 'json not null) default character set utf8mb4' at line 1`

Docker wiki execution:
docker run -d -p 8080:3000 -u="root" --name wiki --restart unless-stopped -e "DB_TYPE=mysql" -e "DB_HOST=SERVER" -e "DB_PORT=3306" -e "DB_USER=wiki" -e "DB_PASS=wiki" -e "DB_NAME=db_wiki" requarks/wiki:2

Can you help me?

Thanks

**Comment by NGPixel** [comment]
Make sure your database is at least MySQL 8.0 or later?

**Comment by InformaticaGuttmann** [comment]
Yes im sure.
[System] [MY-010931] [Server] /usr/sbin/mysqld: ready for connections. Version: '8.0.23'  socket: '/var/lib/mysql/mysql.sock'  port: 3306  MySQL Community Server - GPL.

It's a docker container
 docker run --name=mysql1  --restart unless-stopped -v mysqldb:/var/lib/mysql -d mysql/mysql-server:8.0
